registry.json

```diff
@@ -3350,5 +3350,20 @@
     "telegram": "heliastaking",
     "operatorName": "Miro",
     "github": "heliastaking"
+  },
+  {
+    "ticker": "MSTR",
+    "poolId": "abea730f5c32157c76c876597d0b7093d50754d0274d67c62d7434b9",
+    "social": {
+      "facebook": "",
+      "twitter": "MStakepool",
+      "website": "http://www.maestro-pool.com",
+      "reddit": "https://www.reddit.com/user/MAESTRO-stake-pool",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "MAESTRO_Pool",
```

Comment:
I couldnt find this username on telegram, please can you make sure it is the handle of an individual and not a group or channel?

Reply:
@Vardominator Not sure if you saw this comment too? I still can't see MAESTRO_Pool as a user on TG

Reply:
@Bez625 was just getting to this. You should be able to find `varderes` now!